### DISCUSSION TITLE
Mapped CSV Type Casting

### DISCUSSION THREAD

**Discussion by Libertie** [jgile/vue-csv-import#81] [Ideas]
It would be really useful if this component supported type casting for fields. Something like this:
```
<vue-csv-import
  v-model="csv"
  :fields="{name: {required: false, label: 'Name'}, age: {required: true, label: 'Age', type: 'Number'}}"
>
```
This would then be implemented in the `buildMappedCsv` function to produce a mapped CSV with the correct field types. Currently, all fields are mapped as strings.

@jgile, are you open to pull requests for this feature?

**Comment by James-Burgess** [comment]
PR's are welcome and this sounds like an excellent addition!
Defaulting to the current behaviour or converting to string if no type is passed would be required to not break anything.
Ill be happy to help out if you need any assistance.

**Reply by Libertie** [reply]
I've submitted a pull request. Very open to feedback! I am a newbie when it comes to making contributions on GitHub so I won't be offended if I need to do additional work on this.
